### PR TITLE
Add NumberKnobInput to django-gentelella-widgets

### PR DESCRIPTION
**Pull request by JulioOrdonezV**
first working commit for the numberknubinput widget

### CODE REVIEW

**Comment by JulioOrdonezV** [comment]
Add some fixes to the pull request

File: demo/demoapp/urls.py [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
+from .views import formView
 
 pclss = Personclass()
 countryclss = Countryclass()
 menuclss = MenuItemclass()
 
 urlpatterns = [
-        path('create/notification', create_notification_view)
+        path('create/notification', create_notification_view),
+        path('form', formView),
```
Comment (luisza) [inline demo/demoapp/urls.py:13]: Path needs to be significant, so path('form', ...) is a bad name for this view

File: demo/demoapp/views.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render, redirect
 from djgentelella.notification import create_notification
+from .forms import RawFooForm, FooForm
```
Comment (luisza) [inline demo/demoapp/views.py:5]: Better name for RawFooForm is required

File: demo/demoapp/forms.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class FooForm(CustomForm, forms.ModelForm):
+    class Meta:
+        model = Foo
+        fields = ['number_of_eyes']
+
+class RawFooForm(CustomForm, forms.Form):
+    age = forms.IntegerField(
+            widget=knobwidget.NumberKnobInput(attrs={
```
Comment (luisza) [inline demo/demoapp/forms.py:14]: It's Need test without attrs.

File: demo/demoapp/models.py [hunk 4]
```diff
@@ -16,4 +16,9 @@ class Person(models.Model):
...
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
+
+class Foo(models.Model):
+    age = models.IntegerField()
+    speed_in_miles_per_hour = models.FloatField()
+    number_of_eyes = models.IntegerField()
```
Comment (luisza) [inline demo/demoapp/models.py:24]: Min-Max validator test is required

File: demo/demoapp/views.py (hunk 2 above)
Comment (JulioOrdonezV) [reply demo/demoapp/views.py]: changed to FooModelForm and FooBasicForm

File: demo/demoapp/forms.py (hunk 3 above)
Comment (JulioOrdonezV) [reply demo/demoapp/forms.py]: Added the field without attrs to the FooBasicForm

File: demo/demoapp/urls.py (hunk 1 above)
Comment (JulioOrdonezV) [reply demo/demoapp/urls.py]: Changed it to knobwidget/testform

File: demo/demoapp/models.py (hunk 4 above)
Comment (JulioOrdonezV) [reply demo/demoapp/models.py]: Added validators to the model fields

**Comment by JulioOrdonezV** [comment]
This pull requests adds the numberknobinput widget to django-gentelella, more information about this widget can be found on the original jquery plug-in repository https://github.com/aterrien/jQuery-Knob. Inside the demoapp application there is an example on how to use this plug-in with a ModelForm and a BasicForm. It should be able to handle the same attributes as the original plug-in

**Pull request merged**